### ISSUE TITLE
RabbitMQ issues with traffic and port probes

### ISSUE THREAD

**Issue by Unitech** [keymetrics/pm2-io-apm#152]
There is an issue when patching RabbitMQ with these custom probes:

https://github.com/keymetrics/pm2-io-apm/blob/master/src/metrics/network.ts#L136

**Comment by Unitech** [comment]
From vmarchaud:

```
I will advice you to disable monitoring network in pmx for now, we are wrapping the socket constructor to get those metrics in pure javascript (AMQP client must do the same)
What you could do is use pm2-server-monit to monitor these datas since it use linux binary to get them, it should work well.
```

**Comment by Unitech** [comment]
Can we avoid to patch rabbitmq while keeping this network probe active? @wallet77

**Comment by vmarchaud** [comment]
bump @wallet77

**Comment by wallet77** [comment]
Fixed in 2.3.0

**Comment by numbertank** [comment]
We have found same issue in latest 2.4.5 version
